src/Microsoft.Health.Fhir.Core/Configs/CoreFeatureConfiguration.cs

```diff
@@ -76,5 +76,10 @@ public class CoreFeatureConfiguration
         /// Gets or sets a value indicating whether the server supports the $status operation for SearchParameters.
         /// </summary>
         public bool SupportsSelectableSearchParameters { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the server supports the $bulk-delete.
+        /// </summary>
+        public bool SupportsBulkDelete { get; set; }
```

Comment:
Why do we want this configuration parameter when we have the one in the job host configuration?

Reply:
This blocks the API call and excludes it from the CapabilityStatement. Looking forward I think it still might be easier to delete this setting and clean up the checks after its enabled.